apple/internal/xcframework_rules.bzl

```diff
@@ -1026,6 +1026,75 @@ def _apple_static_xcframework_impl(ctx):
             ),
         )
 
+        # Bundle resources
+        rule_descriptor = rule_support.rule_descriptor_no_ctx(
+            link_output.platform,
+            apple_product_type.framework,
+        )
+        platform_prerequisites = platform_support.platform_prerequisites(
+            apple_fragment = ctx.fragments.apple,
+            config_vars = ctx.var,
+            cpp_fragment = ctx.fragments.cpp,
+            device_families = ctx.attr.families_required.get(
+                link_output.platform,
+                default = rule_descriptor.allowed_device_families,
+            ),
+            disabled_features = ctx.disabled_features,
+            explicit_minimum_deployment_os = ctx.attr.minimum_deployment_os_versions.get(
+                link_output.platform,
+            ),
+            explicit_minimum_os = ctx.attr.minimum_os_versions.get(link_output.platform),
+            features = ctx.features,
+            objc_fragment = ctx.fragments.objc,
+            platform_type_string = link_output.platform,
+            uses_swift = link_output.uses_swift,
+            xcode_version_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig],
+        )
+        resource_deps = _unioned_attrs(
+            attr_names = ["deps"],
+            split_attr = ctx.split_attr,
+            split_attr_keys = link_output.split_attr_keys,
+        )
+        partial_output = partial.call(partials.resources_partial(
+            actions = actions,
+            apple_mac_toolchain_info = apple_mac_toolchain_info,
+            bundle_extension = ".framework",
+            bundle_name = bundle_name,
+            # TODO(b/174858377): Select which environment_plist to use based on Apple platform.
+            environment_plist = ctx.file._environment_plist_ios,
+            executable_name = executable_name,
+            launch_storyboard = None,
+            output_discriminator = library_identifier,
+            platform_prerequisites = platform_prerequisites,
+            resource_deps = resource_deps,
+            rule_descriptor = rule_descriptor,
+            rule_label = label,
+            version = None,
+        ))
```

Comment:
above this is mostly copied from the dynamic xcfw implementation, same with the attrs, the loop below is the unique part